src/pages/Cart/CartItem.tsx

```diff
@@ -59,22 +61,29 @@ const CartItem: React.FC<CartItemProps> = ({ isMobile, data, onRemove, onQuantit
   const subTotalPrice = displayPrice((productInfo?.price ?? 0) * data.quantity);
 
   const quantityInput = (
-    <InputGroup size="xs">
-      <InputLeftAddon style={{ cursor: "pointer" }} onClick={() => handleQtyChangeCounter(false)}>
-        -
-      </InputLeftAddon>
-      <Input
-        type="tel"
-        pattern="[0-9]*"
-        textAlign="center"
-        value={data.quantity}
-        placeholder="Item Count"
-        onChange={handleQtyChangeInput}
-      />
-      <InputRightAddon style={{ cursor: "pointer" }} onClick={() => handleQtyChangeCounter(true)}>
-        +
-      </InputRightAddon>
-    </InputGroup>
+    <Flex flexDirection="column" gap={1}>
+      <InputGroup size="xs">
+        <InputLeftAddon style={{ cursor: "pointer" }} onClick={() => handleQtyChangeCounter(false)}>
+          -
+        </InputLeftAddon>
+        <Input
+          type="tel"
+          pattern="[0-9]*"
+          textAlign="center"
+          value={data.quantity}
+          placeholder="Item Count"
+          onChange={handleQtyChangeInput}
+        />
+        <InputRightAddon style={(data.quantity < MAX_ITEM_CNT) ? { cursor: "pointer" } : { cursor: "not-allowed", opacity: 0.4 }} onClick={() => handleQtyChangeCounter(true)}>
```

Comment:
decided to grey out the "+" button and change the cursor to "not-allowed" if the user tries to increase the quantity beyond the quantity in stock, but we could consider adding a more significant warning/error message? @chanbakjsd

Reply:
I believe this is good enough for now, we can change it in the future if we have a better plan.